sigh/lib/assets/resign.sh

```diff
@@ -201,8 +201,9 @@ then
 fi
 
 # Set the app name
-# The app name is the only file within the Payload directory
-APP_NAME=$(ls "$TEMP_DIR/Payload/")
+# In Payload directory may be another file except .app file, such as StoreKit folder.
+# Search the first .app file within the Payload directory
+APP_NAME=$(ls "$TEMP_DIR/Payload/"|grep ".app$"| head -1)
```

Comment:
BTW; if there's any doubt that the Payload directory might contain other files ending in .app, but not directories, one might want to use instead:
````
APP_NAME=`( cd "$TEMP_DIR/Payload/" ; find . -maxdepth 1 -type d -name "*.app" | head -1 | cut -c 3- ) `
````

Test code:
````
TEMP_DIR=/tmp

mkdir "$TEMP_DIR/Payload"
touch "$TEMP_DIR/Payload/StoreKit"
touch "$TEMP_DIR/Payload/T0.app"
mkdir "$TEMP_DIR/Payload/T1.app"


APP_NAME=$(ls "$TEMP_DIR/Payload/"|grep ".app$"| head -1)
echo "APP_NAME 1 ${APP_NAME}"

APP_NAME=`( cd "$TEMP_DIR/Payload/" ; find . -maxdepth 1 -type d -name "*.app" | head -1 | cut -c 3- ) `
echo "APP_NAME 2 ${APP_NAME}"
```

produces:
````
APP_NAME 1 T0.app
APP_NAME 2 T1.app
````